clusters/hive/pools/cvp/cvp-ocp-4-9-amd64-aws-us-west-2_clusterpool.yaml

```diff
@@ -24,16 +24,16 @@ spec:
     name: install-config-aws-us-west-2
   labels:
     tp.openshift.io/owner: cvp
-  maxSize: 10
+  maxSize: 5
   platform:
     aws:
       credentialsSecretRef:
         name: cvp-aws-credentials
       region: us-west-2
   pullSecretRef:
     name: pull-secret
-  runningCount: 1
-  size: 5
+  runningCount: 0
+  size: 1
```

Comment:
The maintenance support for 4.9 will end in April 18, 2023
* https://access.redhat.com/support/policy/updates/openshift#dates

IMHO the change for 4.9 should be done in few weeks and not now.